### DISCUSSION TITLE
Fatal error

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#268] [ОШБК!] [answered]
Здравствуйте, подскажите пожалуйста. Открываю одну из таблиц выдает ошибку, ее можно как то исправить или удалять и заново создавать таблицу? Fatal error: Allowed memory size of 1073741824 bytes exhausted (tried to allocate 20480 bytes) in /home/totum/totum-mit/totum/tableTypes/aTable.php on line 685

**Comment by ttm-alexeypolunin** [comment, marked as answer]
https://ru.docs.totum.online/debugging

**Comment by Leo-ui-528** [comment]
Спасибо, разобрался
